src/types/smartSearch.ts

```diff
@@ -1,8 +1,3 @@
-import { ZetkinSmartSearchFilter } from './zetkin';
-
-export interface ZetkinSmartSearchFilterWithId extends ZetkinSmartSearchFilter {
-    id: number;
-}
 
 export enum FILTER_TYPE {
     ALL ='all',
```

Comment:
unrelated to review but also wanted to ask, why use enums over plain objects?

Reply:
Enums can be used as types so they provide type checking, useful for situations like this where you wanna use them as constants. Enums, I'm pretty sure, have some syntactic stuff going on to make sure they aren't accidentally mutated, which you wouldn't want for constants. We used to do `Object.freeze({...}) Before which was whack.